browser/src/control/jsdialog/Util.KeyboardGridNavigation.ts

```diff
@@ -164,4 +131,17 @@ function getRowColumn(element: HTMLElement): [number, number] {
 	return [row, column];
 }
 
-JSDialog.MoveFocus = moveFocus;
+JSDialog.KeyboardGridNavigation = function (
+	container: HTMLElement,
+	options?: any,
+) {
+	container.addEventListener('keydown', (event: KeyboardEvent) => {
+		const activeElement = document.activeElement as HTMLElement;
+		handleKeyboardNavigation(
+			event,
+			activeElement,
+			activeElement.parentElement,
+			...options,
```

Comment:
can we explicitly pass the arguments?
it will help to catch mistakes in the future if we use types

Reply:
it's the only call, so it's not a lot of work

Reply:
> can we explicitly pass the arguments?
@eszkadev can you give an example of passing args explicitly ?
I kind of not sure about that.....

Reply:
just don't use `options?: any` - where you put array of params and then you unfold it
please add names and types for all args you pass here:
```
JSDialog.KeyboardGridNavigation = function (
	container: HTMLElement,
        handleColorSelection: ...,
        builder: ...,
        data: ...,
        palette: ...
```